packages/qmongjs/src/components/Layout/Footer.tsx

```diff
@@ -48,6 +48,12 @@ export const Footer: React.FC = () => {
           <Link title="Link til personvern" href={`/personvern`}>
             Personvern
           </Link>
+          <Link
```

Comment:
mangler et linjeskift her (`<br />`).

Reply:
Ok, den så riktig ut når jeg debugget, men de andre har linjeskift, så da burde vel denne også ha det.

Reply:
Kan også slette den til helseatlas hvis de ikke skal har Personvern og Informasjonskapsler. Jeg så at det allerede lå en for personvern der, så derfor la jeg til den for informasjonskapsler der. I tilfelle det skulle legges til i menyen.